senders/msteams/msteams.go

```diff
@@ -0,0 +1,208 @@
+package msteams
+
+import (
+	"bytes"
+	"encoding/json"
+	"errors"
+	"fmt"
+	"github.com/moira-alert/moira"
+	"gopkg.in/russross/blackfriday.v2"
+	"net/http"
+	"net/url"
+	"strings"
+	"time"
+)
+
+// Sender implements moira sender interface via MS Teams
+type Sender struct {
+	frontURI string
+	logger   moira.Logger
+	location *time.Location
+	client   *http.Client
+}
+
+// Init initialises settings required for full functionality
+func (sender *Sender) Init(senderSettings map[string]string, logger moira.Logger, location *time.Location, dateTimeFormat string) error {
+	sender.logger = logger
+	sender.location = location
+	sender.frontURI = senderSettings["front_uri"]
+	sender.client = &http.Client{
+		Timeout:   time.Duration(30) * time.Second,
+		Transport: &http.Transport{DisableKeepAlives: true},
```

Comment:
Why keep alive is disabled here? As I can understand ms teams have only one domain and keeping connect will not raise memory usage depending on amount of alerts.

Reply:
good point!

Reply:
how about we make it configurable?

Reply:
I'm thinking something like this
```
disableKeepAlives, err := strconv.ParseBool(senderSettings["disable_keepalives"])
if err != nil {
	disableKeepAlives = true
}
sender.client = &http.Client{
	Timeout:   time.Duration(30) * time.Second,
	Transport: &http.Transport{DisableKeepAlives: disableKeepAlives},
}
```